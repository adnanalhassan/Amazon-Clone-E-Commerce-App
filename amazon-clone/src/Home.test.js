import Home from "./Home";
import { shallow } from "enzyme";
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
Enzyme.configure({ adapter: new Adapter() });

describe(Home, () => {
     it("renders the home page correctly", () => {
        const wrapper = shallow(<Home />);
        expect(wrapper).toMatchSnapshot();
     });
});