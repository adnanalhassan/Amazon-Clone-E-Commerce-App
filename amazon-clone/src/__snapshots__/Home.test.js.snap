// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders the home page correctly 1`] = `ShallowWrapper {}`;
